### PR TITLE
tour value is now set and retrieved from one place

### PR DESCRIPTION
**Pull request by frimpongopoku**
Tour still works the same way as it used to. 
I made tweaks so that we can 

- [X] Toggle tour on and off via url params 
- [X] Also, so that we set and access the tour value from only one place -- redux. After it has been preloaded from local storage in AppRouter.js 

This helps make testing easier as well! Tour was getting in the way of EVERYTHING! Now we can turn tour off via url when we are running our tests

### CODE REVIEW

**Review by BradHN1 — CHANGES_REQUESTED** [review]
Hi @frimpongopoku - looks good but there are two issues:
First - the "Take Tour" menu option doesn't work now.  I'd like it to go to the homepage and start the tour, if that is possible.  Of course that menu option comes from the back-end, with the source in menus.json;
Second - issue 848 - if a user comes to the site from a page other than the Home page, we don't want the tour to start.  I had coded up a solution to this but not pushed it yet.  My suggestion is that the value of showTour has three options: "true", "false" or "in_progress'  and if you come to the ActionsPage for example, it doesn't show the tour unless it is in the "in_progress" state.

**Comment by frimpongopoku** [comment]
@BradHN1, great thinking 🔥 , I will look into that ASAP

**Comment by frimpongopoku** [comment]
@BradHN1 using `in_progress` got messy really quickly, so to prevent future headaches I took another route 🤣 🤣 🤣 , which involved me adding an id value to the take tour item in the menu on the backend... 🤣 🤣 .  [Here is the pr for that](https://github.com/massenergize/api/pull/436)

File: src/AppRouter.js [hunk 1]
```diff
@@ -89,11 +91,52 @@ class AppRouter extends Component {
...
+    const home = homegroup?.children?.find(homeFxn);
+    var location = this.cleanURL(window.location.href);
+    var rebuilt = this.cleanURL(
+      window.location.protocol + window.location.host + home.link
+    );
+    return location === rebuilt;
   }
 
+  /**
+   * IDEA: The idea is that we use this function to determine the value of the tour
```
Comment (Opoku-Agyemang) [inline src/AppRouter.js:116]: thanks for putting this comment here! Super clear and helpful 🙇🏿

**Review by Opoku-Agyemang — APPROVED** [review]
LGTM

**Pull request merged**